dpgen2/entrypoint/submit.py

```
@@ -81,6 +82,7 @@
     CustomizedLmpTemplateTaskGroup,
     ExplorationStage,
     ExplorationTask,
+    LmpSpinTaskGroup,
```

Ruff

`dpgen2.exploration.task.LmpSpinTaskGroup` imported but unused

Remove unused import

(F401)